test/integration/targets/azure_rm_webapp/tasks/main.yml

```diff
@@ -0,0 +1,171 @@
+- name: Fix resource prefix
+  set_fact:
+    linux_app_plan_resource_group: "{{ (resource_prefix | replace('-','x'))[-10:] }}linuxappplan"
+    win_app_name: "{{ (resource_prefix | replace('-','x'))[-8:] }}winapp"
+    win_plan_name: "{{ (resource_prefix | replace('-','x'))[-8:] }}winplan"
+    linux_plan_name: "{{ (resource_prefix | replace('-','x'))[-8:] }}lnxplan"
+
+- name: 1 - Create a windows web app with non-exist app service plan
```

Comment:
For basic create/update/delete, need to run integration test with check_mode/normal/idempotent 3 scenarios.

Reply:
added delete scenario